examples/cfd/01_convection.ipynb

```diff
@@ -348,19 +351,19 @@
      "name": "stderr",
      "output_type": "stream",
      "text": [
-      "CustomCompiler: compiled /var/folders/xr/858m074j16nd3kcz4dd58_f00000gn/T/devito-5i1151cw/99828d970bd77277b20e4c775f82fd0cecee0f39.c [0.13 s]\n",
+      "CustomCompiler: compiled /var/folders/qr/4hc1nw7d3hj55jbd1z7kkrvh0000gn/T/devito-89_5b_7l/1957c22c54d3d7bfc6d9c309adc532acf2d80e64.c [0.66 s]\n",
       "=========================================================================================\n",
-      "Section section_1<100,80> with OI=0.00 computed in 0.000 s [0.00 GFlops/s]\n",
-      "Section section_2<100,80> with OI=0.00 computed in 0.000 s [0.00 GFlops/s]\n",
-      "Section main<100,80,80> with OI=1.45 computed in 0.001 s [7.25 GFlops/s, 0.60 GPts/s]\n",
+      "Section section_1<101,81> with OI=0.00 computed in 0.000 s [0.00 GFlops/s]\n",
+      "Section section_2<101,81> with OI=0.00 computed in 0.000 s [0.00 GFlops/s]\n",
+      "Section main<101,81,81> with OI=1.50 computed in 0.001 s [11.11 GFlops/s, 0.93 GPts/s]\n",
```

Comment:
The number of iterations has changed here. So has the OI. Neither of those should be expected.

Reply:
in fact, they should (read my general comment below). We haven't changed ``shape`` in ``Grid(shape=...)``, so no offsets in the loop bounds means we get exactly 81 here, no matter what the space order is.

Reply:
What about OI though?

Reply:
+points => +flops, +accesses => different numerator, different denominator => different OI

Reply:
(according to the way the OI is currently computed, see ``profiling.py``)

Reply:
So our OI calculation is dependent on the grid size?